central/image/datastore/store/store.go

```diff
@@ -18,11 +18,11 @@ type Store interface {
 	Exists(ctx context.Context, id string) (bool, error)
 
 	Get(ctx context.Context, id string) (*storage.Image, bool, error)
-	GetMany(ctx context.Context, ids []string) ([]*storage.Image, []int, error)
+	GetByIDs(ctx context.Context, ids []string) ([]*storage.Image, error)
```

Comment:
Could we use variadic arguments here to merge `Get` and `GetByIDs`?

Reply:
Good point I create an issue for this as it will be good to have it in all stores
https://issues.redhat.com/browse/ROX-28998